css/main.css

```diff
@@ -0,0 +1,73 @@
+:root {
+  --soft-blue: hsl(215, 51%, 70%);
+  --cyan: hsl(178, 100%, 50%);
+  background-color: hsl(217, 54%, 11%);
+
+  --card-bg-color: hsl(216, 50%, 16%);
+  --dark-blue-line-color: hsl(215, 32%, 27%);
+}
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+body {
+  width: 100%;
+}
+.main-body {
+  width: 100%;
+}
+.trivia-cards-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  gap: 1rem;
+}
+.main-heading {
+  margin: 1rem;
+  color: var(--white);
+}
+.main-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
+}
+#trivia-card {
+  background-color: var(--card-bg-color);
+  height: 25rem;
+  width: 22rem;
+  border-radius: 1rem;
+  cursor: pointer;
+}
+.card-content-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 100%;
+  padding: 1rem 1rem 1rem 1rem;
```

Comment:
If values are same, you can shorten it to 1 value. i.e. `padding: 1rem;`

Reply:
Sure!